apps/projects/app/components/Content/Issues.js

```diff
@@ -653,18 +652,16 @@ class Issues extends React.PureComponent {
                   <List
                     height={scrollHeight}
                     itemCount={preparedIssues.length}
-                    itemSize={92}
+                    estimatedItemSize={index =>
+                      preparedIssues[index].labels.totalCount > 0 ? 137 : 97
```

Comment:
Whether or not the item is funded also effects that size of the issue, right now adding funding to an issue appears to break the height.

Reply:
Great catches, @Quazia. I also noticed that the height of items wasn't updating when changing the filters or sorting, but I got that working. Check it out:

![VariableSizeList behavior when filtering and sorting](https://user-images.githubusercontent.com/221614/59460502-38219000-8ded-11e9-8dc5-54119045ab57.gif)

---

However, I'm not sure how robust we can make the experience on *mobile*. Here's what it looks like now:

![VariableSizeList behavior on narrow screens](https://user-images.githubusercontent.com/221614/59460501-38219000-8ded-11e9-9862-351e09549770.gif)

---

 I think we have two options for fixing it:

1. Try to reverse-engineer the browser's layout engine for a given item, using the container width + content length to guess at how the content will flow unto multiple lines. I foresee this having endless bugs, and becoming out of date anytime we make subtle styling changes that don't seem immediately related to this list (example: if the next version of aragonUI tweaks what `size="normal"` means)
2. Re-design these cards so that they have a perfectly predictable height

I don't like either of these options, and I wonder if it's best to leave it as-is for now, and circle back to getting this pixel-perfect once we've had more time to play with it.

(But what do I mean by "as-is"? This PR as-is? Or the codebase as-is, without this PR? I haven't experienced the pain that this PR is trying to fix, so I don't know; I'll leave that up to someone else.)

Reply:
Note: I don't think there's an easy to configure `react-window` to update the item height with each item's "natural height". I've asked about this here: https://github.com/bvaughn/react-window/issues/263#issuecomment-501462501

Reply:
The main pain-point this PR is trying to resolve is having to click "show more" instead of having full scrolling for the issues. I think leaving this "as-is" for now is the best course of action as having list items spill over or render with the incorrect height is a bigger bug that this was meant to solve.